src/test/ml-modules/root/test/suites/merging-xml/test-data/custom-sjs-merge-options.xml

```diff
@@ -0,0 +1,39 @@
+<options xmlns="http://marklogic.com/smart-mastering/merging">
+  <match-options>basic</match-options>
+  <thresholds>
```

Comment:
merge options should no longer have thresholds: https://project.marklogic.com/jira/browse/MDM-298